napari/layers/_labels_layer/model.py

```diff
@@ -416,9 +414,9 @@ def fill(self, indices, coord, old_label, new_label):
         new_label : int
             Value of the new label to be filled in.
         """
-        int_coord = copy(coord)
-        int_coord[0] = int(round(coord[0]))
-        int_coord[1] = int(round(coord[1]))
+        int_coord = list(copy(coord))
```

Comment:
`list` already makes a copy

```suggestion
        int_coord = list(coord)
```

Reply:
fixed